src/Catalyst.Core.Lib.Tests/UnitTests/P2P/IO/Messaging/Broadcast/BroadcastHandlerTests.cs

```diff
@@ -0,0 +1,103 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.KeySigner;
+using Catalyst.Abstractions.P2P.IO.Messaging.Broadcast;
+using Catalyst.Core.Lib.IO.Handlers;
+using Catalyst.Core.Lib.Util;
+using Catalyst.Core.Modules.Cryptography.BulletProofs;
+using Catalyst.Protocol.Cryptography;
+using Catalyst.Protocol.Wire;
+using Catalyst.TestUtils;
+using Catalyst.TestUtils.Protocol;
+using DotNetty.Transport.Channels.Embedded;
+using Microsoft.Reactive.Testing;
+using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using Serilog;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.P2P.IO.Messaging.Broadcast
+{
+    public sealed class BroadcastHandlerTests
+    {
+        private readonly IBroadcastManager _fakeBroadcastManager;
+        private readonly BroadcastHandler _broadcastHandler;
+        private readonly IKeySigner _keySigner;
+        private readonly ProtocolMessage _broadcastMessageSigned;
+        private readonly SigningContext _signingContext;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_signingContext' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)